authority/admin.py

```diff
@@ -149,16 +149,21 @@ class PermissionAdmin(admin.ModelAdmin):
 
     def formfield_for_dbfield(self, db_field, **kwargs):
         # For generic foreign keys marked as generic_fields we use a special widget
```

Comment:
I added this as running `tox` locally would fail for `py37-dj22` and report the following:
```
======================================================================
ERROR: test_raw_id_GenericForeignKey (authority.tests.PermissionAdminForDBFieldTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/bia/django-authority/example/../authority/tests.py", line 490, in test_raw_id_GenericForeignKey
    raw_id_fields=['object_id'])
  File "/Users/bia/django-authority/example/../authority/tests.py", line 479, in assertFormfield
    ff = ma.formfield_for_dbfield(model._meta.get_field(fieldname), request=None)
  File "/Users/bia/django-authority/example/../authority/admin.py", line 154, in formfield_for_dbfield
    for f in self.model._meta.virtual_fields
AttributeError: 'Options' object has no attribute 'virtual_fields'
```
[Seems that `virtual_fields` were deprecated in Django 1.10](https://docs.djangoproject.com/en/3.0/releases/1.10/#id3) and that 2.2 only has `private_fields` https://docs.djangoproject.com/en/2.2/_modules/django/db/models/options/